pymemcache/client/base.py

```diff
@@ -1003,19 +1020,17 @@ def _store_cmd(self, name, values, expire, noreply, flags=None, cas=None):
 
             results = {}
             buf = b''
+            line = None
             for key in keys:
-                buf, line = _readline(self.sock, buf)
+                try:
+                    buf, line = _readline(self.sock, buf)
+                except MemcacheUnexpectedCloseError as e:
+                    self.close()
+                    raise e
                 self._raise_errors(line, name)
 
                 if line in VALID_STORE_RESULTS[name]:
-                    if line == b'STORED':
-                        results[key] = True
-                    if line == b'NOT_STORED':
-                        results[key] = False
-                    if line == b'NOT_FOUND':
-                        results[key] = None
-                    if line == b'EXISTS':
-                        results[key] = False
```

Comment:
The last three of these should have been `elif` instead of `if`. I don't think fixing that would have reduce the linter's complexity score though?

I'm actually equally in favor of dropping that lint rule.